test/test.nix

```diff
@@ -19,6 +19,8 @@ import ./make-test.nix rec {
       # hardened
     ];
 
+    virtualisation.diskSize = 1024;
```

Comment:
```suggestion
    # Needed because duplicity has huge temp space requirements (270 MB for 3 MB of backup data)
    virtualisation.diskSize = 1024;
```